tests/console/yast2_apparmor.pm

```diff
@@ -107,6 +115,11 @@ sub run {
     send_key 'alt-n';
     wait_serial("yast2-apparmor-status-0", 200) || die "'yast2 apparmor' didn't finish";
 
+    }
+    else {
+        record_soft_failure 'Managing of profiles through gui currently not implemented';
```

Comment:
record_soft_failure only with bug references

Reply:
There is currently no bug reference until I get a reply in Fate, since the goal is to get this test running again I think we can leave it out until we know if it is expected or not and adapt the soft failure at that moment

Reply:
no bug, no record_soft_failure

Reply:
but, you can add recoed_info with reference to the fate. I doubt that you'll get reply anytime soon.